pkg/database/migrations/31_scenes_captions.up.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE `scene_captions` (
+  `scene_id` integer,
+  `language_code` varchar(255) NOT NULL,
+  `path` varchar(255) NOT NULL,
+  `caption_type` varchar(255) NOT NULL,
+  foreign key(`scene_id`) references `scenes`(`id`) on delete CASCADE
+);
+
+CREATE UNIQUE INDEX `scene_captions_caption_unique` on `scene_captions` (`path`);
```

Comment:
After some thinking i dont think the index is useful. In some corner cases it might cause some fails as well.
For example adding a subtitle for two different scenes eg
`filename.srt` (or filename.en.srt ) for scenes `filename.mp4`, `filename.avi`
The unique restriction should probably be on scene_id, path and i doubt we would ever need it